docs/setup/README.md

```diff
@@ -0,0 +1,334 @@
+# Setup
+
+This document goes through the steps you need to take to set up a token gateway.
+
+## Prerequisites
+
+You need:
+
+* a connection to nodes that are connected to origin and auxiliary (for help see [how to connect]('../test_net/README.md))
+* OST on auxiliary to deploy contracts
+* an EIP20 token deployed on origin
+* a utility token deployed on auxiliary
+* optional: you already have `Organization`s deployed on origin and auxiliary
+* optional: you already have state root providers (e.g. `Anchor`s) deployed on origin and auxiliary
+
+The following is required to use mosaic.js setup:
+
+```js
+const { Setup } = require('@openstfoundation/mosaic.js');
+const Web3 = require('web3');
+
+const origin = {
+  web3: new Web3('http://localhost:8545'),
+  chainId: '3',
+  // The address that you will use to deploy contracts:
+  deployer: '0x6d4e5f5ca54b88fdb68a5d0e6ea5aa3869f88116',
+  txOptions: {
+    gasPrice: '13000000000', // 13 GWei
+  },
+  // The address of your EIP20 token:
+  token: '0x2902a9E437c07168fc73B96821ed82b8eB69B20F',
+  // The address of OST (you can use this one):
```

Comment:
this is on Ropsten, and there is no automated way to request OST on Ropsten.  We need to set up an OST faucet

Reply:
Correct. But it is still right that the user should use our instance of OST, right? They rather require OST on auxiliary in order to facilitate. A faucet there would be more helpful in my opinion.

Reply:
we can provide both

Reply:
Should I mention anything about the current lack of a faucet?

Reply:
OK let's get the faucets rolling on Monday.